bot/seasons/easter/egghead_quiz.py

```diff
@@ -0,0 +1,118 @@
+import asyncio
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'egghead_questions.json'), 'r', encoding="utf8") as f:
+    EGGHEAD_QUESTIONS = load(f)
+
+
+EMOJIS = [
+    '\U0001f1e6', '\U0001f1e7', '\U0001f1e8', '\U0001f1e9', '\U0001f1ea',
+    '\U0001f1eb', '\U0001f1ec', '\U0001f1ed', '\U0001f1ee', '\U0001f1ef',
+    '\U0001f1f0', '\U0001f1f1', '\U0001f1f2', '\U0001f1f3', '\U0001f1f4',
+    '\U0001f1f5', '\U0001f1f6', '\U0001f1f7', '\U0001f1f8', '\U0001f1f9',
+    '\U0001f1fa', '\U0001f1fb', '\U0001f1fc', '\U0001f1fd', '\U0001f1fe',
+    '\U0001f1ff'
+]
+
+
+class EggheadQuiz(commands.Cog):
+    """This cog contains the command for the Easter quiz!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.quiz_messages = {}
+
+    @commands.command(aliases=["eggheadquiz", "easterquiz"])
+    async def eggquiz(self, ctx):
+        """
+        Gives a random quiz question, waits 30 seconds and then outputs the answer
+
+        Also informs of the percentages and votes of each option
+        """
+
+        random_question = random.choice(EGGHEAD_QUESTIONS)
+        question, answers = random_question["question"], random_question["answers"]
+        answers = [(EMOJIS[i], a) for i, a in enumerate(answers)]
+        correct = EMOJIS[random_question["correct_answer"]-1]
+
+        valid_emojis = [emoji for emoji, _ in answers]
+
+        description = "\n".join([f"{emoji} -> **{answer}**" for emoji, answer in answers])
+
+        q_embed = discord.Embed(title=question, description=description, colour=Colours.pink)
+
+        msg = await ctx.send(embed=q_embed)
+        for emoji in valid_emojis:
+            await msg.add_reaction(emoji)
+
+        self.quiz_messages[msg.id] = valid_emojis
+
+        await asyncio.sleep(30)
+
+        del self.quiz_messages[msg.id]
+
+        msg = await ctx.channel.get_message(msg.id)  # Refreshes message
```

Comment:
The pinned d.py version has changed since you created your branch. Please ensure that you've merged in the latest changes from seasonalbot's `master` branch and updated your developer environment. `pipenv sync --dev` should take care of the latter for you. 

`get_message()` no longer exists, it has been replaced by [`fetch_message()`](https://discordpy.readthedocs.io/en/rewrite/api.html#discord.abc.Messageable.fetch_message)